lib/renaissance_web/templates/auction/_details.html.eex

```diff
@@ -0,0 +1,10 @@
+<div class="mod-table">
+  <table class="item-table">
+    <tr class="table-row"><td class="table-section table-left">Price:</td><td class="bold table-section table-right"><%= @current_price %></td></tr>
```

Comment:
there's a lot of class repetition

Reply:
🤔 Hmm... I haven't done a lot of html & css refactoring. Any general tips for cleaning up html & css?

Reply:
You could say all the trs under item table have that styling instead of the particular class